client/src/components/DemoUser/DemoUser.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Typography, Grid } from '@material-ui/core';
+import login from '../../helpers/APICalls/login';
+import useStyles from './useStyles';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import { useAuth } from '../../context/useAuthContext';
+
+// Login credentials for a demo user
+const EMAIL = 'd1@g.com';
+const PASSWORD = '123456';
+
+export default function DemoUser(): JSX.Element {
+  const classes = useStyles();
+
+  const { updateSnackBarMessage } = useSnackBar();
+  const { updateLoginContext } = useAuth();
+
+  const handleLogin = (email: string, password: string) => {
+    login(email, password).then((data) => {
+      if (data.error) {
+        updateSnackBarMessage(data.error.message);
+      } else if (data.success) {
+        updateLoginContext(data.success);
+      } else {
+        // should not get here from backend but this catch is for an unknown issue
+        console.error({ data });
+
+        updateSnackBarMessage('An unexpected error occurred. Please try again');
+      }
+    });
+  };
+  return (
+    <Grid container justify="center" className={classes.addMargin}>
+      <Typography className={classes.demoText}>Want to try it out first?</Typography>
+      <Button
+        onClick={() => handleLogin(EMAIL, PASSWORD)}
```

Comment:
Is the guest user also submitting a username and password? If guest maybe you don't need them :-)

Reply:
No the guest is not expected to submit a username and password. However, according to the [specifications](https://github.com/hatchways/team-brioche/issues/18#issue-992729568), when they click on the `login as a visitor` button,